app/controllers/user.js

```diff
@@ -0,0 +1,34 @@
+const { User } = require('../models');
+const { validateEmail, validatePassword } = require('../helpers/utils');
```

Comment:
Check schema-validator. You can validate the params (query, param, body, etc) of the request. For example, you can check if the body has the email or the name.